contrib/docker/multi-stage-frontend/README.md

```diff
@@ -0,0 +1,19 @@
+# Standalone Dockerfile for frontend
+
+This directory contains the resources which will help you build backstage without any requirements
+other than docker itself. It uses a multi-stage Dockerfile to build and ship backstage.
+
+## Usage
+
+You can simply run the following command to build backstage.
+
+```
+# Make sure you are in the root directory of backstage then run
+docker build -t backstage -f ./contrib/docker/multi-stage-frontend/Dockerfile .
```

Comment:
Maybe have the example tag be `backstage-frontend`

Reply:
Done.